Formula/c/cliproxyapi.rb

```diff
@@ -0,0 +1,36 @@
+class Cliproxyapi < Formula
+  desc "Wrap Gemini CLI, Codex, Claude Code, Qwen Code as an API service"
+  homepage "https://github.com/router-for-me/CLIProxyAPI"
+  url "https://github.com/router-for-me/CLIProxyAPI/archive/refs/tags/v6.1.3.tar.gz"
+  sha256 "3ba69aa7e7af86f0e23230059bfbb01856c5c8b0bece2ec828e31afbe8be687c"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.Version=#{version}
+      -X main.Commit=#{tap.user}
+      -X main.BuildDate=#{time.iso8601}
+      -X main.DefaultConfigPath=#{etc/"cliproxyapi.conf"}
+    ]
+
+    system "go", "build", *std_go_args(ldflags:), "cmd/server/main.go"
+    etc.install "config.example.yaml" => "cliproxyapi.conf"
+  end
+
+  service do
+    run [opt_bin/"cliproxyapi"]
+    keep_alive true
+  end
+
+  test do
+    require "pty"
+    PTY.spawn(bin/"cliproxyapi", "-login", "-no-browser") do |r, _w, pid|
+      sleep 5
+      Process.kill "TERM", pid
+      assert_match "accounts.google.com", r.read
```

Comment:
Sorry, maybe we should use nonblock for linux.
```suggestion
      assert_match "accounts.google.com", r.read_nonblock(1024)
```